cmd/algoh/blockWatcher.go

```diff
@@ -89,6 +91,13 @@ func (bw *blockWatcher) run(watchers []blockListener, stallDetect time.Duration,
 				break
 			}
 
+			// Parse the raw block
```

Comment:
@algochoi Minor and optional:  What do you think of the following refactoring?

Since `client.RestClient` is the provided implementation, I _think_ it's possible to replace `RawBlock` with `EncodedBlockCert` (from `libgoal.go`) + remove the highlighted decoding.

I don't feel strongly - feel welcomed to resolve as is.

Reply:
I considered it, but kept as-is to reduce the diffs as using `libgoal` code introduces more code.

Reply:
@algochoi Got it - Resolving.